### ISSUE TITLE
bump intl to 0.18.0 when it reaches stable channel

### ISSUE THREAD

**Issue by Macacoazul01** [hmkrivoj/month_picker_dialog#57]

**Comment by amigne** [comment]
This becomes a critical requirement:
```
Resolving dependencies...
Because month_picker_dialog 1.0.0 depends on flutter_localizations from sdk which depends on intl 0.18.0, month_picker_dialog 1.0.0 requires intl 0.18.0.
And because month_picker_dialog 1.0.0 depends on intl ^0.17.0, month_picker_dialog 1.0.0 is forbidden.
So, because no versions of month_picker_dialog match >1.0.0 <2.0.0 and timetracker depends on month_picker_dialog ^1.0.0, version solving failed.
Process finished with exit code 1
```

**Comment by Macacoazul01** [comment]
@amigne as stated on the pr related to the issue, as intl 0.18 isn't the official one on stable channel I won't bump it yet.

Just use 

dependency_overrides:
  intl: ^0.18.0

**Comment by Imesh7** [comment]
@hmkrivoj can you assign this issue to me, I will fixed it

**Comment by Macacoazul01** [comment]
For everyone watching this issue, there's a big chance of intl 0.18.0 on stable channel after google io on the next days.

**Comment by Macacoazul01** [comment]
Solved in 2.0
flutter_localizations is currently using intl 0.18.0 instead of 0.18.1, so 0.18.0 will be used here until they bump the version there
